guides/03_building-with-blocks/04_custom-CSS-and-JS.md

```diff
@@ -2,6 +2,8 @@
 
 This guide covers how to style Blocks with more flexibility, as well as adding Javascript code to event listeners. 
 
+**Warning**: Custom JS and CSS is *not* guaranteed to work across Gradio versions as the Gradio HTML DOM may change. We recommend using custom CSS and JS sparingly.
```

Comment:
JS should work across gradio versions - it's specifically query selectors that may break.
```suggestion
**Warning**: The use of query selectors in custom JS and CSS is *not* guaranteed to work across Gradio versions as the Gradio HTML DOM may change. We recommend using query selectors sparingly.